client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -118,14 +119,23 @@ export class CommentNavigation extends Component {
 			toggleBulkMode,
 			unlike,
 		} = this.props;
+		const progressId = uuid();
 		this.props.removeNotice( 'comment-notice' );
 		each( selectedComments, ( { commentId, isLiked, postId, status } ) => {
 			if ( 'delete' === newStatus ) {
-				deletePermanently( postId, commentId );
+				deletePermanently( postId, commentId, {
+					progressId,
+					progressTotal: selectedComments.length,
```

Comment:
This screams for us to just pass a list of comment Ids, but let's perhaps bump priority to that batch endpoint instead. :)

In terms on naming how about `bulkCount`? or something similar, from the current name I assume this means it updates some progresses current total.